react/src/js/react/posts/PostPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import PostFeed from "./PostFeed";
+
+/**
+ * Show the feed and filtering feature for posts
+ */
+export default class PostPage extends React.Component {
```

Comment:
This could just be a stateless component

Reply:
yeah you are right, good enough to merge now?